main/authorization.bs

```diff
@@ -19,3 +21,31 @@ the HTTP `Link` header with a `rel` value of `acl` [[#acl]].
 In the event that a server can't apply an ACL to a resource, it MUST deny
 access.
 [[Source](https://github.com/solid/specification/issues/130#issue-532777017)]
+
+Servers exposing client's access privileges on a resource URL MUST advertise
+by including the `WAC-Allow` HTTP header in the response of HTTP `HEAD` and
+`GET` requests.
+
+The syntax for `WAC-Allow`, using the ABNF syntax is:
+
+```
+WAC-Allow           = "WAC-Allow" ":" OWS permissions OWS
+permissions         = user-permissions OWS "," OWS public-permissions
+user-permissions    = "user=" quoted-access-modes
+public-permissions  = "public=" quoted-access-modes
+quoted-access-modes = DQUOTE *( access-modes / SP ) DQUOTE
```

Comment:
The original code allows for an arbitrary non-zero number of whitespace characters. Access modes MUST be separated by whitespace (not the case here).

Reply:
https://github.com/solid/specification/commit/6d1bc9e1a90f63294e6bc38e6645665e6904121f#diff-6e4282542b3e3fcc1e8048e34c85b6e806de95de91f6ae62d3273d47ec2eaeb6R35 should do it.